content/docs/rfcs/61/README.md

```diff
@@ -0,0 +1,373 @@
+7. Control node sends the magnet link created in step 6 to community members via special channel created in step 3 through the Waku network.
+8. Every subsequent 7 days, steps 6 and 7 are repeated and the new message archive data is appended to the previously created message archive data.
+
+## Serving archives for missed messages
+
+If the control node goes offline (where "offline" means, the control node's main process is no longer running), it MUST go through the following process:
+
+1. Control node restarts
+2. Control node requests messages from store nodes for the missed time range for all channels in their community
+3. All missed messages are stored into control node's local message database
+4. If 7 or more days have elapsed since the last message history torrent was created, the control node will perform step 6 and 7 of [Serving community history archives](#serving-community-history-archives) for every 7 days worth of messages in the missed time range (e.g. if the node was offline for 30 days, it will create 4 message history archives)
+
+## Receiving community history archives
+
+Community member nodes go through the following (high level) process to fetch and restore community message histories:
+
+1. User joins community and becomes community member (see [org channels spec](https://github.com/status-im/specs/pull/151))
+2. By joining a community, member nodes automatically subscribe to special channel for message archive metadata exchange provided by the community
+3. Member node requests live message history (last 30 days) of all the community channels including the special channel from store nodes
+4. Member node receives waku message ([14/WAKU2-MESSAGE](https://rfc.vac.dev/spec/14/)) that contains the metadata magnet link from the special channel
+5. Member node extracts the magnet link from the waku message and passes it to torrent client
+6. Member node downloads [message archive index](#message-history-archive-index) file and determines which message archives are not downloaded yet (all or some)
+7. Member node fetches missing message archive data via torrent
+8. Member node unpacks and decompresses message archive data to then hydrate its local database, deleting any messages for that community that the database previously stored in the same time range as covered by the message history archive
+
+# Storing live messages
+
+For archival data serving, the control node MUST store live messages as [14/WAKU2-MESSAGE](https://rfc.vac.dev/spec/14/).
+This is in addition to their database of application messages. 
+This is required to provide confidentiality, authenticity, and integrity of message data distributed via the BitTorrent layer, and later validated by community members when they unpack message history archives.
+
+Control nodes SHOULD remove those messages from their local databases once they are older than 30 days and after they have been turned into message archives and distributed to the BitTorrent network.
```

Comment:
Yes, they will still seed.
Generally, control nodes always seed one torrent per community. The torrent itself contains the entire data.
If the messages have been removed from the local database, they still exist in the archive data in the torrent.
That's why, in fact, a control node needs to be able to store a message thrice: once as `WakuMessage`, once as unwrapped application message (so it can be rendered on screen), and once as part of the archive in the torrent.

Only thing to note here is, if the Waku messages are removed from the database, the control node cannot recreate archives for those, unless it's able to recover those messages from store nodes (which is unlikely if the messages are older than 30 days).

Hence, the spec says "SHOULD", as it's more memory efficient, but it comes at a cost of course.